training/cambricon/baichuan2_13b-deepspeed/config/requirements.txt

```diff
@@ -0,0 +1,5 @@
+regex
+schedule
+accelerate
+transformers
```

Comment:
Recommended but not compulsory, use transformers==\<version\> to avoid exceptions in the later test.

Reply:
have fixed it.